man/systemd-journal-upload.service.xml

```diff
@@ -45,7 +45,10 @@
     Unless limited by one of the options specified below, all journal entries accessible to the user
     the program is running as will be uploaded, and then the program will wait and send new entries
     as they become available.</para>
-
+    
+    <para><command>systemd-journal-upload</command> transfers the raw content of journal file and
+    uses HTTP as a transport protocol. It is not compatible with syslog.</para> 
```

Comment:
no need to specify incompatibilities, there's no claim of compatibility anywhere

Reply:
Updated.